package.json

```
@@ -20,6 +20,7 @@
   "scripts": {
     "dependencies": "yarn",
     "size": "size-limit",
+    "watch": "tsc-watch",
```

added this in to aid with the typescript workflow

---

it looks like we don't use anything special in tsc-watch
```suggestion
    "watch": "tsc --watch",
```

---

😅  👍